project/Dependencies.scala

```diff
@@ -51,6 +52,7 @@ object Dependencies {
     "com.univocity"                  % "univocity-parsers"   % "2.4.1",
     "org.ocpsoft.prettytime"         % "prettytime"          % "4.0.1.Final",
     "org.everit.json"                % "org.everit.json.schema" % "1.4.1",
+    "com.github.pathikrit"          %% "better-files"        % "2.17.1",
```

Comment:
I'd love to upgrade to 2.12 but unfortunately there's no spray build for 2.12, so upgrading to akka-http is a dependency..